libphobos/libdruntime/core/threadasm.S

```diff
@@ -495,3 +504,64 @@ _fiber_switchContext:
     // 'return' to complete switch
     ret;
 #endif
+/************************************************************************************
+ * i386- and x86_64-apple-darwin POSIX ASM BITS
+ ************************************************************************************/
+// if POSIX boils down to this (reference http://nadeausoftware.com)
+#if !defined(_WIN32) && (defined(__unix__) || defined(__unix) || (defined(__APPLE__) && defined(__MACH__)))
+.text
+.p2align 4
```

Comment:
Switched to .p2align because .align arg is treated differently by GAS on different systems.  I knew I ran into that once long ago, something was nagging me while out and about today.